mayavi/tests/test_sources.py

```diff
@@ -15,7 +15,7 @@ def test_convert_to_arrays(self):
         """Test if the convert_to_arrays routine works properly"""
 
         values = convert_to_arrays(
-            (1, 2, 3, np.float(4), np.float64(5), np.array([6])))
```

Comment:
Deal with NumPy deprecating `np.float` as it's just an alias of the builtin `float`. So here I've changed the meaning to be `np.float32`, but this seemed more appropriate than having two `float64`s in there (not 100% sure what the motivation was for the original test).